src/test/java/com/wcc/platform/service/CmsServiceTest.java

```diff
@@ -71,18 +70,18 @@ void whenGetFooterGivenValidJson() throws IOException {
 
   @Test
   void whenGetCollaboratorGivenInvalidJsonThenThrowsInternalException() throws IOException {
```

Comment:
throws IOException should be replaced with "throws JsonProcessingException" across all tests methods